spec/lib/coach/request_serializer_spec.rb

```diff
@@ -34,5 +35,39 @@
         expect(sanitized).to eq('value')
       end
     end
+
+    context "as an instance" do
+      let(:mock_request) do
+        instance_double("Request", :request, format: nil,
+                                             remote_ip:           nil,
+                                             uuid:                nil,
+                                             method:              nil,
+                                             filtered_parameters: nil,
+                                             filtered_env:        {
+                                               "foo" => "bar",
+                                               "HTTP_bar" => "bar"
+                                             })
+      end
+
+      subject(:request_serializer) { described_class.new(mock_request) }
+
+      describe "#serialize" do
+        it "serializes a request" do
+          allow(mock_request).to receive(:fullpath) { raise }
+
+          serialized = request_serializer.serialize
+
+          expect(serialized[:path]).to eq('unknown')
+        end
+
+        it "rescues any exceptions request#fullpath may raise" do
+          allow(mock_request).to receive(:fullpath) { raise }
```

Comment:
I believe we favour the `.and_raise` over `{ raise }` in tests. It may also be cleaner to move `serialized` into the subject for this `describe` block, and then reduce this test to `allow`ing the raise and verifying the path value. @rthbound what do you think?